iac-extensions/docker/src/test/resources/checks/SpaceBeforeEqualInKeyValuePairCheck/SpaceBeforeEqualInKeyValuePairCheck.dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM scratch
+
+ARG WITH_EQUAL="=val"
+ARG JUST_EQUAL==
+ARG WITHOUT_EQUAL="val"
+
+
+# Non compliant use cases
+# Noncompliant@+1 {{Remove space before the equal sign in the key-value pair, as it can lead to unexpected behavior.}}
+LABEL VERSION =42
+#     ^^^^^^^^^^^
+# Noncompliant@+1
+LABEL VERSION = 42
+# Noncompliant@+1
+LABEL VERSION =42 NAME=BOB
```

Comment:
Will it work the other way around, `LABEL VERSION=42 NAME =Bob`? Or would that be invalid Dockerfile?

Reply:
This is indeed an invalid docker syntax and also crash our parser, so that's why I didn't add it :grin: